core/iwasm/aot/aot_runtime.h

```diff
@@ -88,6 +88,7 @@ typedef struct AOTFunctionInstance {
 } AOTFunctionInstance;
 
 typedef struct AOTModuleInstanceExtra {
+    const uint32 *stack_sizes;
```

Comment:
Had better use `DefPointer(const uint32 *, stack_sizes)`? Suppose the aot code may access more fields here in the future, we can ensure the offsets of these fields are fixed, and are the same in compilation time and execution time.

Reply:
ok

Reply:
done